lib/widgets/screens/channel_messages/channel_messages.dart

```diff
@@ -117,17 +136,16 @@ class _ChannelChatState extends State<ChannelChat> {
   final ScrollController listScrollController = ScrollController();
   final Duration _animationDuration = const Duration(milliseconds: 250);
   ChatModel? _chatModel;
+  MessagesProvider? _messagesProvider;
```

Comment:
You may mark it as `late final` and skip the `?`.

Reply:
You're super right! Replaced this and other declarations to use lazy initializer.